starknet-foundry/crates/cast/src/main.rs

```diff
@@ -79,7 +84,8 @@ async fn main() -> Result<()> {
 
     let accounts_file_path =
         Utf8PathBuf::from(shellexpand::tilde(&cli.accounts_file_path).to_string());
-    if !&accounts_file_path.exists() {
+    if let Commands::Account(_) = &cli.command {
+    } else if !&accounts_file_path.exists() {
         bail! {"Accounts file {} does not exist! Make sure to supply correct path to accounts file.", cli.accounts_file_path}
     }
```

Comment:
I'd just create this file here, if it does not exist.
Or even better - it would be cool to create the file if the flag is of default value, and error out if someone passes custom path (not sure if it is possible though)

Reply:
I'm not sure if we should create this file here. It would apply to every command and if someone is using cast for calls only it is unwanted behaviour

Reply:
Yeah, I thought so as well. Maybe wqe could approach it somewhat different, like adjusting `declare` to instruct to create/add account to file, if the file does not exist? I wonder if, in that scenario, we should count on user to make it work, or try and write some kind of helper for this. What are your thoughts, gents?

Reply:
btw we could also add this info, if file exists but account does not exist in this file, probably

Reply:
Ok I understand now what's happening here and I think that the logic for the account file is split (here we check for existence but only if the command's not `account` and in `starknet_commands::account::create::create` we do the rest of the stuff). I do not think this is good.

I think you do not have to declare `accounts_file_path` here. You can just define a method that lazily does it or create a lazy static for this. What do you think?